src/Scale_identification.jl

```diff
@@ -0,0 +1,100 @@
+using DelimitedFiles
+using Statistics
+using StatsBase
+using MIDI
+using MusicManipulations
+
+"""
+    MIDI_to_notes
+
+Returns the pitch of an array of MIDI note value, inside of single octave.
+
+Example :
+julia> Midi_to_notes([0,13,26])
+["C","C","C"]
+
+"""
+function MIDI_to_notes(MIDInotes::Array{})
+    name = ["C", "C#", "D", "D#", "E", "F", "F#", "G", "G#", "A", "A#", "B"]
+    MIDIrange = []
+    for i in range(-1,12)
```

Comment:
Are you working on Julia version 1.0.0 or greater? We don't support earlier Julia versions and this piece of code is not valid there: `range(-1, 12)`.

Reply:
I am working on julia 1.0, this is the function Base.range. should I change it ?